receptorctl/receptorctl/cli.py

```diff
@@ -487,6 +494,12 @@ def submit(
                 allparams.append(params["params"])
             allparams.extend(cmdparams)
             params["params"] = " ".join(allparams)
+        
+        # Process environment variables
+        if envvars:
+            envvar_dict = dict(s.split("=", 1) for s in envvars)
```

Comment:
Do we want to enforce the "=" in the key, value pair for envvars?

Reply:
Thanks a mill, 

Please note this whoever takes this receptorctl work on